### ISSUE TITLE
NoMethodError: undefined method `type` for NilClass

### ISSUE THREAD

**Issue by itsderek23** [scoutapp/scout_apm_ruby#110]
A customer is reporting the following error (screenshot from Rollbar):

![](https://downloads.intercomcdn.com/i/o/28927463/f2d2fb7b1066e0f72f32636a/image.png)

I haven't yet determined if this is happening on every request or just a specific request.

I believe this is running Rails 5.0.4 and Ruby 2.2.0.

**Comment by cschneid** [comment]
I believe this was fixed with: https://github.com/scoutapp/scout_apm_ruby/commit/259959b4e184b41a158ecaa5141c829706601959

It is not yet rolled out into a released agent. It will go out with the next version.

**Comment by IgorPetkovic** [comment]
@cschneid Hey, do you know if the release is coming soon ? Because this is causing issues with Active Admin for us. I'd have to remove the gem if it wont be soon.

**Comment by itsderek23** [comment]
@IgorPetkovic - we're checking it out this week. 

@dlanderson - mind cutting a release?

As a workaround, you can grab this fix by using the master branch of the agent:

```
# Gemfile
gem 'scout_apm', git: 'https://github.com/scoutapp/scout_apm_ruby.git'
```

**Comment by dlanderson** [comment]
Hi @IgorPetkovic - I've released this fix in 2.1.27. Sorry for the delay!

**Comment by IgorPetkovic** [comment]
@itsderek23 @dlanderson Hey, great news! Thanks for the quick replies and release :)
